state/executor.go

```diff
@@ -1183,6 +1184,11 @@ func (t *Transition) GetRefund() uint64 {
 	return t.state.GetRefund()
 }
 
+// ActivePrecompiles returns addresses of precompile contracts
+func (t *Transition) ActivePrecompiles() []types.Address {
+	return t.precompiles.Addrs
```

Comment:
I think we need to return only active precompiles based on active forks:

```suggestion
	 return t.precompiles.ActivePrecompiles(&t.config)
```

Plus, you would need to add the same function in the `precompiled.go`:

```go
// ActivePrecompiles returns a list of active precompiled contract addresses based on the provided configuration.
// It checks if each contract address in the precompiled contracts map can be run with the given configuration.
// If a contract address can be run, it is added to the list of active precompiles.
// The list of active precompiled contract addresses is returned.
func (p *Precompiled) ActivePrecompiles(config *chain.ForksInTime) []types.Address {
	var precompiles []types.Address

	for addr := range p.contracts {
		if p.CanRun(&runtime.Contract{CodeAddress: addr}, nil, config) {
			precompiles = append(precompiles, addr)
		}
	}

	return precompiles
}
```

Reply:
Not need the function eventually. Marking as fixed.